common/src/main/java/org/conscrypt/AddressUtils.java

```diff
@@ -56,7 +56,7 @@ public static boolean isValidSniHostname(String sniHostname) {
     /**
      * Returns true if the supplied hostname is an literal IP address.
      */
-    public static boolean isLiteralIpAddress(String hostname) {
+    static boolean isLiteralIpAddress(String hostname) {
```

Comment:
I think this will break testing in Android because the test-class is in a different ClassLoader than the class-under-test.